### ISSUE TITLE
Artifacts with TextureFormat:Compressed and AlphaBlending:true

### ISSUE THREAD

**Issue by PodeCaradox** [MonoGame/MonoGame#6985]
Hello,

i found the following bug if i use the following Options in the Content Pipline
![grafik](https://user-images.githubusercontent.com/5760157/72167129-d4833380-33ca-11ea-8665-9ed3a9746eb5.png)
![grafik](https://user-images.githubusercontent.com/5760157/72167424-64c17880-33cb-11ea-904a-77a1c60cdd83.png)

if i use instead this options(TextureFormat:Color):
![grafik](https://user-images.githubusercontent.com/5760157/72167489-7d319300-33cb-11ea-8bf0-82e43fca2675.png)
![grafik](https://user-images.githubusercontent.com/5760157/72167541-92a6bd00-33cb-11ea-9d13-e39f31a17a21.png)

Texture I'm using but made it smaller for Github
![tile_land_macros1](https://user-images.githubusercontent.com/5760157/72167791-2082a800-33cc-11ea-81fa-416d0c641c13.png)

How can i get Compression without the Artifacts?

#### What version of MonoGame does the bug occur on:
- MonoGame 3.7

#### What operating system are you using:
- Windows

#### What MonoGame platform are you using:
- DesktopGL, Content Pipeline

**Comment by tomspilman** [comment]
That artifact seems a bit odd...  but `Compression` for textures uses a lossy compression format.  So artifacts could occur.

@Jjagg @harry-cpp - I've been wanting to see about doing a simple zip compression for uncompressed color formats.  Like thinking we can always force the XNB to be compressed for Textures with color formats.  This should be an easy win for users.

**Comment by PodeCaradox** [comment]
@tomspilman So there is no way around than use Color? But why is it working when i begin my Spritebatch with BlendState.NonPremultiplied, are the artifacts comme from the Premultiply Alpha?

**Comment by tomspilman** [comment]
> But why is it working when i begin my Spritebatch with BlendState.NonPremultiplied, are the artifacts comme from the Premultiply Alpha?

Could be possible that is a bug...  not sure.  Even with compression we should be obeying the pre-multiply alpha option on the content.  Could be we're not doing that correctly?

**Comment by PodeCaradox** [comment]
Maybe yes, i can make a Test Project if someone want to test it.

**Comment by Jjagg** [comment]
@tomspilman there is the flag at the mgcb level to do lzma compression on everything, but it's not well documented and I've seen a lot of people that don't know about it. Maybe we should enable that flag by default. Decompression is really fast and it can save a bunch of space.

**Comment by tomspilman** [comment]
> there is the flag at the mgcb level to do lzma compression on everything

Yeah that is the flag i meant.  I'm just thinking maybe we should force that flag on a per-XNB basis and in particular for Color textures.

**Comment by Jjagg** [comment]
> but i would hate to break existing ones.

Yeah, definitely agreed.
